cfme/networks/provider/nuage.py

```diff
@@ -153,3 +156,30 @@ def num_network_router(self):
     def num_network_router_ui(self):
         view = navigate_to(self, "Details")
         return int(view.entities.summary("Relationships").get_text_of("Network Routers"))
+
+    @variable(alias="db")
+    def num_cloud_network(self):
```

Comment:
**required** comment out not yet operational variants to avoid confusion

Reply:
Ha, didn't realize I can. Thanks for pointing out, done. I just removed the db variant, no need to have it in as a comment.